src/app/components/MediaPlayer/AudioLoader/index.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import '@testing-library/jest-dom/extend-expect';
+
+import AudioLoader from '.';
+
+describe('Audio Loader', () => {
+  shouldMatchSnapshot(
+    'should match snapshot',
+    <AudioLoader isLoading>Content</AudioLoader>,
+  );
+
+  it('should transclude the children', () => {
```

Comment:
a word I remember from angular!

Reply:
I had this unit test wrapped in an `<ng-app>` but eslint said it was unnecessary